src/gui/advancedsettings.cpp

```diff
@@ -382,10 +379,6 @@ void AdvancedSettings::loadAdvancedSettings()
     checkBoxOsCache.setChecked(session->useOSCache());
     addRow(OS_CACHE, (tr("Enable OS cache") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#disk_io_write_mode", "(?)"))
             , &checkBoxOsCache);
-    // Guided read cache
-    checkBoxGuidedReadCache.setChecked(session->isGuidedReadCacheEnabled());
```

Comment:
checkBoxGuidedReadCache is defined in header, you should remove it too.

Reply:
Oops... I couldn't have guessed that. Why doesn't it have `m_` prefix?..

Reply:
legacy code...

Reply:
Fixed.